pygmt/src/grdimage.py

```diff
@@ -172,6 +157,13 @@ def grdimage(self, grid, **kwargs):
     """
     kwargs = self._preprocess(**kwargs)
 
+    # Do not support -A option
+    if any(kwargs.get(arg) is not None for arg in ["A", "img_out"]):
```

Comment:
```suggestion
    if any(kwargs.get(arg) for arg in ["A", "img_out"]):
```

Should be ok to remove `is not None`, e.g.:

```python
kwargs = {"img_out": "temp.nc"}
if any(kwargs.get(arg) for arg in ["A", "img_out"]):
    raise Exception
```
should raise an Exception.

Reply:
Won't raise an exception if an empty string is given:
```
kwargs = {"img_out": ""}
if any(kwargs.get(arg) for arg in ["A", "img_out"]):
    raise Exception
```

Reply:
Ah yes, keep it as is then.